src/common/lib/client/realtime.ts

```diff
@@ -55,18 +54,34 @@ class Realtime extends Rest {
 class Channels extends EventEmitter {
   realtime: Realtime;
   all: Record<string, RealtimeChannel>;
-  inProgress: Record<string, RealtimeChannel>;
 
   constructor(realtime: Realtime) {
     super();
     this.realtime = realtime;
     this.all = Object.create(null);
-    this.inProgress = Object.create(null);
     realtime.connection.connectionManager.on('transport.active', () => {
       this.onTransportActive();
     });
   }
 
+  channelSerials(): { [name: string]: string } {
+    let serials: { [name: string]: string } = {};
+    for (const [name, channel] of Object.entries(this.all)) {
```

Comment:
`Object.entries` would be a breaking change for IE11 and below. I'd suggest using:
```ts
for (const name of Utils.keysArray(this.all, true)) {
    const channel = this.all[name];
    // ...
}
```

Reply:
:+1: (or `name in this.all` should be fine given it's a null-prototype ob)

Reply:
thanks updated